### ISSUE TITLE
Failed to import IPS Terminology RF2 file

### ISSUE THREAD

**Issue by tmitanitky** [IHTSDO/snowstorm#478]
Hello,

Following "[Loading SNOMED CT into Snowstorm](https://github.com/IHTSDO/snowstorm/blob/master/docs/loading-snomed.md)" , I tried to import IPS Terminology RF2 File (downloaded from [here](https://www.value.snomed.org/international-patient-summary-terminology)) to Snowstorm via Swagger API without success.
This is the log. [snowstorm.log](https://github.com/IHTSDO/snowstorm/files/10682713/snowstorm.log)

[Lower elasticvs.save.batch-size](https://github.com/IHTSDO/snowstorm/issues/244) did not solve the problem.
It was not completely the same but simillar error occured.
This is the log. [snowstorm2.log](https://github.com/IHTSDO/snowstorm/files/10682842/snowstorm2.log)

My goal is
- create translation for IPS Terminology using [Reference set and Translation Service](https://refset.ihtsdotools.org/#/directory).
- export translation and create extension refset file with [empty snapshot](https://github.com/IHTSDO/snowstorm/issues/274)
- import translated IPS Terminology on Snowstorm

Is there a way to resolve this issue?
I am new to SNOMED-CT and wondering I'm doing something fundamentally wrong.
Any help is welcome.

Thanks.

My environments:
Windows 10 Pro
Core i5 7600K
24GB RAM, allocate max 16GB on wsl
Docker Desktop 7.15.0 with wsl2, vm.max_map_count=262144
Snowstorm v8.0.0
IPS Terminology：SnomedCT_IPSTerminologyRelease_PRODUCTION_20221130T120000Z.zip

**Comment by tmitanitky** [comment]
Now I realize I used Snowstorm 8.0.0.
I'm trying Snowstorm 7.12.0.

**Comment by kaicode** [comment]
Thanks for sharing that this import is not working as expected. I don't expect there to be any significant difference between Snowstorm 8.0.0 and 7.12.0. 

It looks the `elasticvs.save.batch-size` took effect because the batch size in the second log is down to 500 documents. If your Elasticsearch instance can not save 500 documents within 30 seconds then try reducing this number further, perhaps 100 would work.

The Elasticsearch write performance is limited by the write speed of the physical disk. A fast SSD is nice to have. I'm not sure if Docker is having any impact here. Once the import is complete the data could be migrated to a lower cost machine with a slower disk.

**Comment by tmitanitky** [comment]
>The Elasticsearch write performance is limited by the write speed of the physical disk. A fast SSD is nice to have. 

Thank you! That was the cause. The storage is HDD.
Finally I succeeded with elasticvs.save.batch-size=20 after tried elasticvs.save.batch-size=100 and failed.

I'm going to try it on aother machine with ssd as well. Thank you.

**Comment by kaicode** [comment]
I'm glad you solved it in the end.
Thank you for sharing your experience, I'm sure someone in the community will benefit from your findings.
